### DISCUSSION TITLE
Generating response curves & mROAS for all channels and storing the values

### DISCUSSION THREAD

**Discussion by isaacmalka** [google/meridian#373] [Q&A]
Hi, 
We'd like to convert the response curve data to dataframe as well as the mROAS data for possible later visualization. Do we have to choose the spend multipliers or is there a default option where get the required data (like in the attached SS)
```
model_medeff = visualizer.MediaEffects(meridian,by_reach=False)
model_medeff.plot_response_curves()
```

To store the data, I wanted to utilize the Analyzer object and store all the data to produce these nice visualizations down the line. 
![Screenshot 2024-11-26 at 2 58 11 PM](https://github.com/user-attachments/assets/0b53b03d-98e9-4205-9002-e9960f74969f)

**Comment by brendalewisprice** [comment]
Yes, you can directly output the data used to create any of the default plots in Meridian. 

For accessing the data used for the response curve, call for the dataframe instead of the plot:
```
model_medeff = visualizer.MediaEffects(meridian, by_reach=False)
response_curves_dataframe = model_medeff.response_curves_data()
```
The mROAS summary data can be output in a similar manner: 
```
media_summary = visualizer.MediaSummary(meridian)
summary_table = media_summary.summary_table()
```
